kas/common.yml

```diff
@@ -4,10 +4,9 @@ header:
 
 repos:
```

Comment:
There should be a info in the main README.md on how to build images with the news kas files. I'm also in favour of keeping the build modular and [passing multiple `yml` files at once](https://kas.readthedocs.io/en/2.1.1/userguide.html#including-configuration-files-via-the-command-line).

Reply:
This could also be combined with not having a specific yaml for targets but pass them through `--target` flag as described in https://github.com/zarhus/zarhus-issues/issues/10#issuecomment-2270813452

Reply:
For now handled with separate `.yml` files. The `--target` parts is TBD. Related commit - https://github.com/zarhus/meta-zarhus/pull/18/commits/67c2b22fe583981582124bb9a7284d83c7105cef